agent/ebs/watcher_windows.go

```diff
@@ -0,0 +1,21 @@
+//go:build windows
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ebs
+
+const (
+	hostMountDir = "C:\\var\\lib\\kubelet"
```

Comment:
@jiuchoe4 - Can you fix this path as part of your testing?

Reply:
Changed and merged this as a part of commit [82fc629](https://github.com/aws/amazon-ecs-agent/commit/82fc62937a73c3e30ea3549112e6b592807955e0)